<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qsynth - A fluidsynth Qt GUI Interface.

   Copyright (C) 2003-2007, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

</comment>
 <class>patchesDialog</class>
 <widget class="QDialog" name="patchesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Qsynth: Channel Preset</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QSplitter" name="m_splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="m_bankListView" >
      <property name="minimumSize" >
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>80</width>
        <height>32767</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Bank selector</string>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="indentation" >
       <number>4</number>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights" >
       <bool>true</bool>
      </property>
      <property name="itemsExpandable" >
       <bool>false</bool>
      </property>
      <property name="sortingEnabled" >
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus" >
       <bool>true</bool>
      </property>
      <column>
       <property name="text" >
        <string>Bank</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="m_progListView" >
      <property name="toolTip" >
       <string>Program selector</string>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="indentation" >
       <number>4</number>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights" >
       <bool>true</bool>
      </property>
      <property name="itemsExpandable" >
       <bool>false</bool>
      </property>
      <property name="sortingEnabled" >
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus" >
       <bool>true</bool>
      </property>
      <column>
       <property name="text" >
        <string>Patch</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Name</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>120</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_okButton" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="iconSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancelButton" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="iconSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <tabstops>
  <tabstop>m_okButton</tabstop>
  <tabstop>m_cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
